pkg/options/options.go

```diff
@@ -59,6 +60,10 @@ func FromEnv(withMachine bool) (*Options, error) {
 	if err != nil {
 		return nil, err
 	}
+	retOptions.StartupScript, err = fromEnvOrError("STARTUP_SCRIPT")
```

Comment:
I think you can make this optional by just reading `os.Getenv("STARTUP_SCRIPT")`. Right now it would fail if it's empty